### ISSUE TITLE
more options for MP3 playback

### ISSUE THREAD

**Issue by TharathielCB** [Polprzewodnikowy/N64FlashcartMenu#83]
### Description

Hi,

I tested the mp3 player right now, which is really cool. 
Could you add some more playback options like **play the whole folder**, shuffle/random and repeat?

Thanks :)

**Comment by Polprzewodnikowy** [comment]
Suggested features are possible and rather easy to implement. MP3 player is not a priority right now so I won't touch it soon. However, I encourage other people to try implementing these features.

**Comment by jesusburnsneon** [comment]
Thanks for creating this, @TharathielCB.  I agree.  Even just adding the ability to play the next track (or go to the previous track) on the MUSIC PLAYER without having to go back to the song list, select the track, and then play it.  If implemented by someone else, I think it's worth considering the use of L and R buttons for this purpose.

![Screenshot 2024-05-08 19-52-53 ed](https://github.com/Polprzewodnikowy/N64FlashcartMenu/assets/73132118/614d8a80-e45b-4def-abf8-086071ecebae)

**Comment by eightbit0675** [comment]
I have to agree here. While I realize the MP3 player was probably thrown in as an afterthought as it is not what the primary function of the SC is supposed to be for, it is incredibly amazing and useful.

When not playing games I like the fact that I can browse around my large MP3 collection and play them here. I have no other consoles that can do that, and this works so incredibly well. Its fast, error free as far as I can tell and just a joy to use.

While "random, repeat, shuffle" would all be nice, what it really needs first are the basic queueing functions. To be able to queue a song to play next, or a whole folder to play next, and to be able to save that playlist for later. And as mentioned to be able to forward and back through tracks in a folder using the L/R buttons. In my humble opinion those features alone would take this player to the next level. You already have an outstanding player that navigates quickly through folders and plays tunes instantly with seemingly no issue. 

I hope somebody will take on the task to improve it as it really deserves the treatment. And, I want to say that I am REALLY grateful for what we have here already so please do not take what I am saying the wrong way :)

**Comment by networkfusion** [comment]
#368

**Comment by networkfusion** [comment]
Released.
